systemtest/src/main/java/io/strimzi/systemtest/utils/StUtils.java

```diff
@@ -373,4 +377,26 @@ public static boolean isParallelNamespaceTest(ExtensionContext extensionContext)
                 .toLowerCase(Locale.ENGLISH)
                 .contains(PARALLEL_NAMESPACE)).count() == 1;
     }
+
+    /**
+     * Copies the image pull secret from the default namespace to the specified target namespace.
+     * @param namespace the target namespace
+     */
+    public static void copyImagePullSecret(String namespace) {
+        LOGGER.info("Checking if secret {} is in the default namespace", SYSTEM_TEST_STRIMZI_IMAGE_PULL_SECRET);
+        if (kubeClient("default").getSecret(SYSTEM_TEST_STRIMZI_IMAGE_PULL_SECRET) == null) {
+            throw new RuntimeException(SYSTEM_TEST_STRIMZI_IMAGE_PULL_SECRET + " is not in the default namespace!");
+        }
+        Secret pullSecret = kubeClient("default").getSecret(SYSTEM_TEST_STRIMZI_IMAGE_PULL_SECRET);
```

Comment:
Is the namespace name the same for openshift and kubernetes?

Reply:
I am not sure if I understand your question. Is the default namespace "default" not suitable for openshift?

Reply:
It will work. My question was if kubernetes and openshift will always contain `default` namespace and according to some docs, it should so I think we are fine with this.